frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -151,3 +152,13 @@ export type NodeProvider = (string) => ComponentType<NodeProps>;
 export type EdgeProvider = (string) => ComponentType<EdgeProps>;
 
 export type GroupProvider = (string) => ComponentType<GroupProps>;
+
+export enum KINDS {
+  DEPLOYMENT = 'Deployment',
+  DEPLOYMENTCONFIG = 'DeploymentConfig',
+  DAEMONSET = 'DaemonSet',
+  STATEFULSET = 'StatefulSet',
+  ROUTE = 'Route',
+  SERVICE = 'Service',
+  BUILDCONFIG = 'BuildConfig',
```

Comment:
nit: We really should be referring to the actual models and using their string constants.

Reply:
@christianvogt you mean remove the enum and use string constants from the modals itself?

Reply:
fixed in https://github.com/openshift/console/pull/2162/commits/7fe748eb664dd4d26deff8e8be7ca583fc832b35